pmd-core/src/main/java/net/sourceforge/pmd/renderers/internal/sarif/SarifLog.java

```diff
@@ -621,6 +638,14 @@ public Integer getPriority() {
             return this.priority;
         }
 
+        /**
+         * A set of distinct strings that provide additional information. This is SARIF 2.1.0 Schema.
+         */
+        @java.lang.SuppressWarnings("all")
+        public String[] getTags() {
```

Comment:
Please don't use arrays anywhere, replace that with `Set<String>`

Reply:
Done 😄